docs/prerequisites/feature-flagging/randomization-sdk/server-sdks/go.md

```diff
@@ -0,0 +1,117 @@
+import Tabs from '@theme/Tabs';
+import TabItem from '@theme/TabItem';
+
+# Go
+
+Eppo's Node SDK is open source:
+- [GitHub repository](https://github.com/Eppo-exp/go-server-sdk)
+- [Package](https://pkg.go.dev/github.com/Eppo-exp/golang-sdk)
+
+### 1. Install the SDK
+You can install the SDK with:
```

Comment:
In your go.mod, add the SDK package as a dependency:

```go
require (
	github.com/Eppo-exp/golang-sdk latest
)
```

Or you can install the SDK from the command line with:

```bash
go get github.com/Eppo-exp/go-server-sdk@latest
```